x-pack/plugins/secops/public/pages/home/index.tsx

```diff
@@ -55,8 +73,26 @@ export const HomePage = pure(() => (
           sort={sort}
           width={900}
         />
-        <WhoAmI sourceId="default">{({ appName }) => <h1>Hello {appName}</h1>}</WhoAmI>
-      </EuiPageContent>
-    </EuiPageBody>
-  </EuiPage>
+      </Divide>
+    </PageContent>
+    <Footer>
+      <WhoAmI sourceId="default">{({ appName }) => <h1>Hello {appName}</h1>}</WhoAmI>
+    </Footer>
+  </PageContainer>
 ));
+
+const Footer = styled.div`
+  position: fixed;
+  left: 0;
+  bottom: 0;
+  width: 100%;
+  border-top: 1px solid #d9d9d9;
+  color: #666;
+  padding: 8px 8px;
+  text-align: center;
```

Comment:
This might be a temporary footer and if so ignore this comment. But this `border-top` + `width:100%`

causes the border to extend all the way to the left cutting through the toolbar on the left. If this is temporary footer you can optionally add a comment about this to line 83 that this is marked for removal and is temporary only for now.

Just pointing this out in case you didn't notice it and don't want it for now:

<img width="1272" alt="screen shot 2018-11-05 at 8 53 14 pm" src="https://user-images.githubusercontent.com/1151048/48042118-60e9f480-e13d-11e8-9121-10145d2c269d.png">